app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,56 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit, update)
+  before_action :valid_user, only: %i(edit, update)
+  before_action :check_expiration, only: %i(edit update)
```

Comment:
1. Combine line 2, 3 and 4 into 1 as they have same actions
2. Add a new line before the actions